packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -84,17 +81,22 @@ const PlansIntervalToggle: React.FunctionComponent< ToggleHostProps > = ( {
 					onClick={ () => onChange( 'ANNUALLY' ) }
 				>
 					<span className="plans-interval-toggle__label">
-						{ __( 'Annually', __i18n_text_domain__ ) }
+						{ __( 'Pay annually', __i18n_text_domain__ ) }
 					</span>
-					{ intervalType === 'MONTHLY' && (
+					{ /*
+					 * Check covers both cases of maxMonthlyDiscountPercentage
+					 * not being undefined and not being 0
+					 */ }
+					{ intervalType === 'MONTHLY' && maxMonthlyDiscountPercentage && (
 						<PopupMessages>
 							{ sprintf(
-								// Translators: "%s" is a number, and "%%" is the percent sign. Please keep the "%s%%" string unchanged when translating.
+								// Translators: will be like "Save up to 30% by paying annually..."
+								// Please keep "%%" for the percent sign
 								__(
-									'Save up to %s%% by paying annually and get a free domain for one year',
+									'Save up to %(maxDiscount)d%% by paying annually and get a free domain for one year',
 									__i18n_text_domain__
 								),
-								maxSavingsPerc
+								{ maxDiscount: maxMonthlyDiscountPercentage ?? 0 }
```

Comment:
Seems like we're checking for `maxMonthlyDiscountPercentage` above, so it will never be `null` / `undefined`, therefore there should be no need for the nullish coalescing I think?